lib/components/form/batch-settings.tsx

```diff
@@ -56,35 +72,84 @@ export function setModeButtonEnabled(enabledKeys: string[]) {
 function BatchSettings({
   activeSearch,
   currentQuery,
+  departArrive,
   enabledModeButtons,
   fillModeIcons,
+  homeTimezone,
   modeButtonOptions,
   onPlanTripClick,
   openAdvancedSettings,
+  routingQuery,
   setQueryParam,
-  spacedOutModeSelector
+  sort,
+  syncSortWithDepartArrive,
+  updateItineraryFilter
 }: Props) {
   const intl = useIntl()
 
-  // Whether the date/time selector is open
-  const [dateTimeOpen, setDateTimeOpen] = useState(false)
-
   // @ts-expect-error Context not typed
   const { ModeIcon } = useContext(ComponentContext)
 
   const processedModeButtons = modeButtonOptions.map(
     pipe(addModeButtonIcon(ModeIcon), setModeButtonEnabled(enabledModeButtons))
   )
 
+  const _planTrip = useCallback(() => {
+    alertUserTripPlan(intl, currentQuery, onPlanTripClick, routingQuery)
```

Comment:
Is this needed? If you click Plan Trip with the origin or destination box empty, you get the message to fill those fields twice.

Reply:
Why did I do this.....